core/src/main/scala/filodb.core/metadata/Dataset.scala

```diff
@@ -88,6 +89,8 @@ object Dataset {
                                     asInstanceOf[SortKeyHelper[K]]
       case java.lang.Double.TYPE => (new DoubleKeyHelper(options.segmentSize.toDouble)).
                                     asInstanceOf[SortKeyHelper[K]]
+      case StringClass         => (new StringKeyHelper(options.segmentSize.toInt)).
+                                    asInstanceOf[SortKeyHelper[K]]
     }
```

Comment:
Hrm, trying to envision why these StringKeyHelper, DoubleKeyHelper, LongKeyHelper need to be asInstanceOf-ed when they are already of type SortKeyHelper